fieldservice/models/fsm_category.py

```diff
@@ -1,30 +1,32 @@
-# Copyright (C) 2018 - TODAY, Open Source Integrators
+# Copyright (C) 2020 Open Source Integrators
```

Comment:
Keep the 2018 copyright. It is important.

Reply:
Ok, I will improve it based on previous Copyright (Copyright (C) 2018 - TODAY, Open Source Integrators)